test/common/algos.mocha.coffee

```diff
@@ -518,12 +518,12 @@ describe 'User', ->
       expect(user.items.hatchingPotions).to.eql {}
       expect(user.items.food).to.eql {}
 
-    it 'gets honey', ->
+    it 'gets red candy', ->
```

Comment:
No, it sounds like you've got the concept right! I should be able to do something like the above.